src/libmongoc/tests/mock_server/mock-server.h

```diff
@@ -64,9 +69,13 @@ void
 mock_server_remove_autoresponder (mock_server_t *server, int id);
 
 int
-mock_server_auto_ismaster (mock_server_t *server,
-                           const char *response_json,
-                           ...);
+mock_server_auto_hello_callback (mock_server_t *server,
```

Comment:
IIUC this won't be used outside of mock-server.c. I think this can be removed from the header and marked as static.